libraries/lib-wave-track/WaveTrack.h

```diff
@@ -199,10 +199,20 @@ class WAVE_TRACK_API WaveChannelInterval final
    const size_t miChannel;
 };
 
+struct WaveTrackMessage {
```

Comment:
I don't expect this to convey more than just the insertion of a new clip, so I think something like `WaveTrackNewClipMessage` would be more informative.

Reply:
Ignore this commit